### ISSUE TITLE
Second camera

### ISSUE THREAD

**Issue by ArieR1963** [occ-ai/scoresight#8]
I want to start using the software for korfball and then I have a scoreboard and a shot clock, only they don't hang together. 
The scoreboard is straight across from me, but the shot clock is on the right side of the room. 
Is it possible to add a second camera?

**Comment by jf-blanco** [comment]
You can merge the two inputs into a scene or one input in obs or vmix and then send that merged signal to scoresight

**Comment by ArieR1963** [comment]
Yes that is a possibility, I now use a solution what is built with Java and in it I can choose a camera for scoreboard and one for shot clock. The shot clock I have to shoot from a different angle, so has a different skew

<img width="774" alt="Screenshot 2024-05-17 at 23 30 13" src="https://github.com/occ-ai/scoresight/assets/44031563/2b9407d3-244b-4964-a029-0e12bd41ff6e">
<img width="660" alt="Screenshot 2024-05-17 at 23 30 39" src="https://github.com/occ-ai/scoresight/assets/44031563/4a56b270-0b68-4589-8ca7-8f13a1d20b11">
<img width="464" alt="Screenshot 2024-05-17 at 23 30 58" src="https://github.com/occ-ai/scoresight/assets/44031563/37d0bb4f-abe0-4395-acec-832fbec83f7b">

**Comment by royshil** [comment]
@ArieR1963 you can run two instances of scoresight at the same time, each looking at a different camera
there's no limit on the number of scoresight you run at once
